### PR TITLE
SPU2: Clean up + add Gaussian value in comment

### PR DESCRIPTION
**Pull request by RedDevilus**
Cleanup some comments, clang format, change copyright date headers and add Gaussian in the comments it did list the other ones.

### CODE REVIEW

File: pcsx2/SPU2/Mixer.cpp [hunk 1]
```diff
@@ -468,21 +468,21 @@ static __forceinline s32 GetVoiceValues(V_Core& thiscore, uint voiceidx)
 
 	switch (InterpType)
 	{
-		case 0:
```
Comment (lightningterror) [inline pcsx2/SPU2/Mixer.cpp:471]: Formatting.

File: pcsx2/SPU2/Mixer.cpp [hunk 2]
```diff
@@ -592,26 +592,26 @@ static __forceinline StereoOut32 MixVoice(uint coreidx, uint voiceidx)
 
 			switch (Interpolation)
 			{
-				case 0:
```
Comment (lightningterror) [inline pcsx2/SPU2/Mixer.cpp:595]: Ditto.

File: pcsx2/SPU2/SndOut_Portaudio.cpp [hunk 3]
```diff
@@ -197,78 +197,78 @@ class Portaudio : public SndOutModule
 			int speakers;
 			switch (numSpeakers) // speakers = (numSpeakers + 1) *2; ?
 			{
-				case 0:
```
Comment (lightningterror) [inline pcsx2/SPU2/SndOut_Portaudio.cpp:200]: Here as well, there may be more, haven't checked.

**Comment by refractionpcsx2** [comment]
> Though does this not need a case 5?

Could do

**Review by tellowkrinkle — COMMENTED** [review]
Any reason for adding periods to the end of comments?  Personally don't think they help anything

**Comment by RedDevilus** [comment]
I've also changed the naming for the interpolation values.

PR:
image

Master:
image

**Comment by RedDevilus** [comment]
Had to resize Interpolation to fit it perfectly.
image

**Review by GovanifY — APPROVED** [review]
Could have been helpful to split the comments/formatting and the UI changes but whatever honestly, LGTM

**Pull request merged**